### ISSUE TITLE
No support for AliasProperty or column mapping

### ISSUE THREAD

**Issue by v-tbert** [EliteLoser/MergeCsv#1]
In the case where you have two objects with a column of matching data that has different name, it doesn't appear you can merge them. Two options would be to support a mapping parameter where you indicate IP on one maps to IPAddress on another, for example. The other would be to support alias properties that were added via Add-Member, for example:
` $NoIpAddressColumn | Add-Member -MemberType AliasProperty -Name IPAddress -Value IP
`
I think the latter may be able to be accomplished by simply adding it to all your get-member calls:
`Get-Member -MemberType NoteProperty,AliasProperty`

**Comment by EliteLoser** [comment]
Sorry about the late reply, I didn't notice before.

To keep it safer with regards to backwards compatibility, I will add a non-default switch parameter called -IncludeAliasProperty and make the proper Get-Member calls use an array that normally only contains "NoteProperty", but will also contain AliasProperty if the switch is passed (if this actually works, it should).

This way people can choose to include the alias properties optionally (in case it breaks someone with weird objects today).

Regarding column mapping, I think Select-Object and the example in my wiki is sufficient, e.g.:
```
$WithIP = $NoIPAddressColumn | Select -Prop *, @{ Name = "IPAddress"; Expression = { $_.IP } }
```

... or similar, maybe just certain properties in addition to the alias property.

I think the world gets it in both bag and sack this way, as the Norwegian saying goes...

**Comment by EliteLoser** [comment]
Now the -IncludeAliasProperty parameter is included in version 1.7. See this link: 
https://www.powershelladmin.com/wiki/Merge_CSV_files_or_PSObjects_in_PowerShell#Alias_Property_Handling
